ARCHITECTURE.md

```diff
@@ -228,10 +228,12 @@ Spanner implements some key features in the
 
 ### Storage
 
-The Storage `*Connection` classes are called `storage::internal::RawClient`,
-which sadly forces our users to reach into the `internal` namespace to mock
-things. Furthermore, some of the `RawClient` decorators should be called
-`*Stub`.
+In Storage the `*Connection` classes are in the `storage::internal` namespace.
+Which sadly forces our users to reach into the `internal` namespace to mock
```

Comment:
s/\. Which/, which/

Reply:
Done.  Also some slight edits if you want to look again.